comid/psareferencevalue.go

```diff
@@ -0,0 +1,64 @@
+package comid
+
+import (
+	"fmt"
+)
+
```

Comment:
Perhaps this comment/question is applicable on a much more top level:

I was thinking, will it be more logically convenient to implement 
(a) Base CoMID and Base CoRIM under CoMID and CoRIM golang Packages

Any PSA specific Profile be extended inside a PSA package, outside of base packages.
Would like to discuss this aspect in our code walkthrough today to hear other opinions on the same!

Reply:
https://github.com/veraison/veraison/issues/65 tracks the issue!